CyganovGS/lab_3/lab3.cpp

```diff
@@ -0,0 +1,51 @@
+#include<iostream>
+#include<string.h>
+#include<stdio.h>
+#include<conio.h>
+
+using namespace std;
+
+int main(){
+	char str_base[256], str_otv[256];
+	int c=0, size_base, i=1, p=0, n_sim=0, j=0, kolvo=0;
+	int znach[255];
+	cout<<"Print your text here, but use only english letters: ";
+	gets(str_base);
+	char array[128][256];
+	size_base=strlen(str_base);
+	while(c<size_base){
+		n_sim=0;
+		znach[kolvo]=0;
+		while(((int)(str_base[c])>=65 && (int)str_base[c]<=90) || (int)((str_base[c])>=97 && (int)(str_base[c])<=122)){
+			array[kolvo][n_sim]=array[kolvo][n_sim]+str_base[c];
```

Comment:
Problem with visual studio appears, because here you use unitialised value of array[kolvo][n_sim]. So, compiller doesn't have to fill the memory with zeros, that's why i saw bad output.